crates/bevy_pbr/src/lib.rs

```diff
@@ -336,7 +336,7 @@ impl Plugin for PbrPlugin {
         app.world.resource_mut::<Assets<StandardMaterial>>().insert(
             Handle::<StandardMaterial>::default(),
             StandardMaterial {
-                base_color: LegacyColor::rgb(1.0, 0.0, 0.5),
+                base_color: Color::srgb(1.0, 0.0, 0.5),
```

Comment:
```suggestion
                base_color: Color::DEEP_PINK,
```
Maybe use `DEEP_PINK` which is almost the same as this one?
`DEEP_PINK: Srgba = Srgba::new(1.0, 0.078, 0.576, 1.0);`